scss/patterns/_footer.scss

```diff
@@ -35,22 +30,37 @@ footer.global {
   } // legal
 
   h2 {
-    font-size: 0.75em;
+    font-size: .75em;
     line-height: 1.4;
     margin-bottom: 0;
-    padding-bottom: 0.5em;
+    padding-bottom: .5em;
   }
 
-  h2, h2 a:link, h2 a:visited {
+  h2, 
+  h2 a:link, 
+  h2 a:visited {
     color: $cool-grey;
     font-weight: normal;
   }
 
-  nav ul li h2 a:after { content: ""; }
+  nav ul li h2 a:after { content: ''; }
```

Comment:
@deadlight From here to line 102 wondering if there is a way we could clean up the nesting and remove some un-required repetition?